### ISSUE TITLE
Support custom homeservers

### ISSUE THREAD

**Issue by ara4n** [matrix-org/matrix.to#15]
with a `?hs=matrix.org` style URL or whatever, as per https://github.com/matrix-org/synapse/issues/1527

**Comment by danger89** [comment]
I agree, this should be the basic core feature for years..

**Comment by almereyda** [comment]
As a side note, supporting custom home servers might also involve supporting custom Element instances.

This idea might also indirectly related to https://github.com/matrix-org/matrix-doc/pull/2312

**Comment by danger89** [comment]
@almereyda That is true, but only valid for desktop Element client I think, for web element it should be rather easy to add support in matrix.to.
